pkg/apis/command/types.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2024 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+const (
+	// ResourceKindReschedule is kind name of Reschedule.
+	ResourceKindReschedule = "Reschedule"
```

Comment:
The Kind name might need to be changed, I don't have a good idea, `TriggerSchedule`?

Reply:
good idea, or I may prefer `ScheduleTrigger`